### ISSUE TITLE
Combined download button not working in public share 14.0.0 Beta 2

### ISSUE THREAD

**Issue by s644** [nextcloud/server#10536]
### Steps to reproduce
1. Create public share link with at least 2 files
2. open public link
3. tick single file and hit Actions/Download Button for combined download

### Expected behaviour
.zip file should be served

### Actual behaviour
page reloads

### Server configuration

**Operating system**:
OMV 4

**Web server:**
nginx
**Database:**
mysql
**PHP version:**
php7
**Nextcloud version:** (see Nextcloud admin page)
14.0.0 Beta 2 (also with Beta 1)
**Updated from an older Nextcloud/ownCloud or fresh install:**
Updated from 13
**Where did you install Nextcloud from:**
Internal Updater

#### Browser log
<details>
<summary>Browser log</summary>

```
js failure:

TypeError: self.fileMultiSelectMenu is null

in file   /apps/files/js/filelist.js:350:5

```
</details>

**Comment by MartB** [comment]
Was broken by https://github.com/nextcloud/server/commit/45db89f3e6b7b391db5f9c766f3b3ef8b4e0750a

**Comment by stratege1401** [comment]
your missing the 14-feedback-beta ...

**Comment by rullzer** [comment]
Fixed with #10537
